sdk/core/azure-core/tests/test_messaging_from_dict.py

```diff
@@ -0,0 +1,44 @@
+import logging
+import sys
+import os
+import pytest
+import json
+
+from azure.core.messaging import CloudEvent
+from devtools_testutils import AzureMgmtTestCase
+from _mocks import (
+    cloud_storage_dict,
+    cloud_custom_dict_base64,
+    cloud_custom_dict_with_extensions,
+)
+
+# Cloud Event tests
+def test_cloud_storage_dict():
+    event = CloudEvent.from_dict(cloud_storage_dict)
+    assert event.data == {
+        "api":"PutBlockList",
+        "client_request_id":"6d79dbfb-0e37-4fc4-981f-442c9ca65760",
+        "request_id":"831e1650-001e-001b-66ab-eeb76e000000",
+        "e_tag":"0x8D4BCC2E4835CD0",
+        "content_type":"application/octet-stream",
+        "content_length":524288,
+        "blob_type":"BlockBlob",
+        "url":"https://oc2d2817345i60006.blob.core.windows.net/oc2d2817345i200097container/oc2d2817345i20002296blob",
+        "sequencer":"00000000000004420000000000028963",
+        "storage_diagnostics":{"batchId":"b68529f3-68cd-4744-baa4-3c0498ec19f0"}
+    }
+    assert event.specversion == "1.0"
+    assert event.__class__ == CloudEvent
+
+
+def test_cloud_custom_dict_with_extensions():
+    event = CloudEvent.from_dict(cloud_custom_dict_with_extensions)
```

Comment:
I actually find it confusing to have the body in another file here. Could we get rid of the _mock.py file and insert the dict here?